.storybook/manager-head.html

```diff
@@ -3,4 +3,11 @@
   .docblock-argstable code {
     white-space: pre-wrap !important;
   }
+
+  /* make the props table category title better readable; align with props table header */
+  .docblock-argstable-body tr[title] button + span {
+    font-size: 13px;
```

Comment:
[optional suggestion] Making this a little larger to stand out a tad more from the rest of the prop names might help folks who are skimming/scrolling quickly!
```suggestion
    font-size: 14px;
```

Reply:
Why I kept it at `13px` was because the rest of the table headers have `13px` and it might be weird if lower hierarchy element have a visually higher hierarchy position? 🤔 (it's only `1px` difference I know but it's off 😅 )

![Screenshot 2024-03-14 at 10 42 07](https://github.com/elastic/eui/assets/44670957/87c96833-4310-4473-92c0-1dfbf62bb879)

I think we might need to increase the `font-size` for all prop table headers and categories to make it feel cohesive?

Reply:
Ah gotcha, I was thinking the text was adding as a header of sorts (although that's not technically/semantically true). I'm not super married to the suggestion, I'd be fine with just leaving it as it is!